src/calendar/date/CalendarUpdateDistributor.js

```diff
@@ -145,7 +145,17 @@ export class CalendarMailDistributor implements CalendarUpdateDistributor {
 		sendMailModel.setBody(body)
 		this._sendStart()
 		await sendMailModel.send(method)
-		                    .finally(() => this._sendEnd())
+		                   .catch((e) => {
+			                   // we remove the attachment from the model to prevent adding more than one calendar file
+			                   // in case the user changes the event and tries to send again a new attachment is created
+			                   sendMailModel.getAttachments().find(file => {
+			                   	if (file && file.name === inviteFile.name){
+			                   		sendMailModel.removeAttachment(file)
```

Comment:
This shouldn't happen inside the call to `find`, move it outside

Reply:
Yes that's true. I changed it.